source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -13,9 +13,9 @@ Retrieve Data
 Overview
 --------
 
-You can perform find operations to retrieve data from your MongoDB database.
-You can perform a find operation to match documents on a set of criteria
-by calling the ``Find()`` method.
+In this guide, you can learn how to use the {+driver-long+} to retrieve
+data from your MongoDB collections by using read operations. Call the
```

Comment:
Suggestion:

I think that using the plural explains the general idea, but the actual usage is limited to a single collection, so I would make sure that part is clear.

"from a MongoDB collection"

Applies to all the descriptions.